include/syscalls.h

```diff
@@ -108,6 +108,7 @@
 	\
 	ID(sys_ioctl) \
 	ID(sys_utimes) \
+	ID(sys_futimes) \
```

Comment:
maybe we can remove `sys_utimes` now that it's implemented in userspace?

Reply:
I will do it in separate pull request to not break libphoenix building.

Reply:
ack